install.sh

```diff
@@ -8,12 +8,13 @@ export USE_DEVELOPMENT_BRANCHES=false
 #     Make pihole scripts fail searching for `systemctl`,
 # which fails pretty miserably in docker compared to `service`
 # For more info see docker/docker issue #7459
-mv "$(which systemctl)" /bin/no_systemctl && \
+which systemctl && mv "$(which systemctl)" /bin/no_systemctl
 # debconf-apt-progress seems to hang so get rid of it too
-mv "$(which debconf-apt-progress)" /bin/no_debconf-apt-progress
+which which debconf-apt-progress && mv "$(which debconf-apt-progress)" /bin/no_debconf-apt-progress
 
 # Get the install functions
-wget -O "$PIHOLE_INSTALL" https://raw.githubusercontent.com/pi-hole/pi-hole/${CORE_TAG}/automated%20install/basic-install.sh
+CUSTOM_INSTALL='development'
+wget -O "$PIHOLE_INSTALL" https://raw.githubusercontent.com/pi-hole/pi-hole/${CUSTOM_INSTALL:-$CORE_TAG}/automated%20install/basic-install.sh
```

Comment:
when is CUSTOM_INSTALL going to be unset seems like the "or core_Tag" bit is uncessary? or is it a temporary thing?

Reply:
It was temporary for the release candidate tag that got deleted, I'll push a refactor that makes it change CORE_TAG based off USE_DEVELOPMENT_BRANCHES